src/main/scala/nl.knaw.dans.easy.dd2d/mapping/Description.scala

```diff
@@ -42,4 +42,16 @@ object Description extends BlockCitation {
     m.addPrimitiveField(DESCRIPTION_VALUE, s"$prefix: ${ node.text }")
     m.toJsonObject
   }
+
+  def newlineToHtml(description: String): String = {
+    //OS agnostic newline and paragraph regex
+    val newline = "\r\n|\n|\r"
+    val paragraph = "(\r\n){2,}|\n{2,}|\r{2,}"
+    description
+      .trim
+      .split(paragraph)
+      .map(p => s"<p>$p</p>")
+      .map(_.replaceAll(newline, "<br>"))
```

Comment:
@ChrisMuller80 : does EASY also preserve single newlines like this? I thought it only converted blank lines to paragraphs. Please verify.

Reply:
@janvanmansum : I checked by looking at the page source of datasets in easy.
e.g.
1. for newlines see: view-source:https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:206964
2. for paragraph see: view-source:https://easy.dans.knaw.nl/ui/datasets/id/easy-dataset:131490
 
I created my own implementation. 
Note: more than two newlines are reduced to a single paragraph.